foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchConfig.java

```diff
@@ -35,6 +35,7 @@
     @NotNull
     @JsonProperty
     private String cluster;
+    private String tableNamePrefix = "foxtrot";
```

Comment:
existing code which creates index prefix looks like this 

return String.format("%s-%s-%s-", ElasticsearchUtils.TABLENAME_PREFIX, table, ElasticsearchUtils.TABLENAME_POSTFIX);

Followed similar convention and named it as table name prefix.